style.css

```
@@ -21,16 +21,23 @@ body {
 	box-sizing: border-box;
 }
 
-
 /*Header*/
 
 .header {
+	position: fixed;
+	width: 100%;
+	background-color: #fff;
+	padding: 8px 0 8px;
```

Já que o valor do top e do bottom são iguais, dá pra simplificar:
```suggestion
	padding: 8px 0;
```